frontend/src/site/login.css

```diff
@@ -0,0 +1,52 @@
+.login {
+    width: 100vw;
+    height: 100vh;
+}
+
+.login form {
+    display: block;
+    border-radius: 5pt;
+    width: 90vw;
+    margin: 20vh auto 10vh auto;
+    box-shadow: 0 0 2px 2px var(--defaultColor);
+    height: 180pt;
+}
+
+.login form div {
+    margin: 5pt;
+}
+
+.login form div label, .login form div input {
```

Comment:
consider using classes instead of general sibling selector - more robust when you change your markup